cloud.foundry.cli/src/test/java/cloud/foundry/cli/operations/SpaceDevelopersOperationsTest.java

```diff
@@ -127,7 +128,7 @@ public void testAssignSpaceDeveloper_ThrowException() {
     }
 
     @Test
-    public void testRemoveSpaceDeveloper() throws InvalidOperationException {
+    public void testRemoveSpaceDeveloper() throws InvalidOperationException, UpdateException {
```

Comment:
This could also be avoided when using RuntimeException as super class for UpdateException

Reply:
Removed.